src/app/welcome/welcome.component.scss

```diff
@@ -65,7 +65,6 @@ header {
   font-family: AeonikMedium;
   font-size: 24px;

```

Comment:
Should add reference to variable font-size from variable.css here (as this font size is being reused several places). but this can be done with the SCSS cleanup-issue (will therefore not request change on this PR).